src/open_inwoner/openklant/tests/data.py

```diff
@@ -38,10 +39,13 @@ def setUpServices(cls):
 
 class MockAPIReadPatchData(MockAPIData):
     def __init__(self):
+        User.objects.filter(email="old@example.com").delete()
```

Comment:
These delete()'s feel a bit kludgey. Are we sure this is needed or if we can fix the tests that want this?

Reply:
I removed them and fixed the issues for the tests that needed it